packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -144,3 +154,12 @@ export interface CategoryFacetBreadcrumb {
   path: CategoryFacetValue[];
   deselect: () => void;
 }
+
+const logClearBreadcrumbs = () =>
```

Comment:
Add a small JSDoc comment here, following the format of other analytics actions.